rskj-core/build.gradle

```diff
@@ -62,6 +62,7 @@ ext {
             spongyCastleVer        : '1.52.0.0',
             bouncyCastleVer        : '1.59',
             ethereumLeveldbJniVer  : '1.18.3',
+            rocksdbJniVer  : '6.27.3',
```

Comment:
minor thing.. could you please align this value with the other version values to fit one column? and the one agains `rocksdbJniLib` below ? )

Reply:
Done